pkgs/development/tools/misc/sqitch/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, writeScript, makeWrapper, sqitchModule }:
+stdenv.mkDerivation rec {
+  name = "sqitch-0.999";
+  buildInputs = [ makeWrapper ];
+  propagatedNativeBuildInputs = [ sqitchModule ];
+  builder = writeScript (name + "-builder.sh") ''
```

Comment:
I have removed the unnecessary propagatedNativeBuildInputs line in my latest commit.

Regarding PERL5LIB: it is not set within the stdenv.mkDerivation builder, so my invocation of wrapProgram has no access to it. It must be deduced based on the ${sqitchModule}/nix-support/propagated-native-build-inputs file. If there's a better way, please let me know.